packages/@glimmer/validator/lib/validators.ts

```diff
@@ -125,24 +123,24 @@ class MonomorphicTagImpl<T extends MonomorphicTagTypes = MonomorphicTagTypes> {
       this.lastChecked = $REVISION;
 
       try {
-        let { subtags, subtag, subtagBufferCache, lastValue, revision } = this;
+        let { subtag, subtagBufferCache, lastValue, revision } = this;
```

Comment:
don't read subtagBufferCache and lastValue here, they aren't used.  They are only used in one spot, just use this.subtagBufferCache and this.lastValue

Reply:
👍